vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -178,5 +189,16 @@ def parse_base_item(item)
         :resource_version   => item.metadata.resourceVersion
       }
     end
+
+    HUMAN_SIZE_SUFFIXES = %w(Ki Mi Gi Ti)
+
+    def human_to_value(value)
+      exp_index = HUMAN_SIZE_SUFFIXES.index(value[-2..-1])
+      if exp_index.nil?
+        return Integer(value)
+      else
+        return Integer(value[0..-3]) * 1024**(exp_index + 1)
```

Comment:
Well, they're not obscure, they're alternate.  Ki ([Kibibyte](http://en.wikipedia.org/wiki/Kibibyte)) is there because of the differences when doing "kilo" which is classically, mutiplying by 1000, vs a computer/base2 "kilo" which is multiplying by 1024.  See also http://en.wikipedia.org/wiki/Kilobyte#1024_bytes

Regardless, I'm ok with this for now, but I think this needs to move into a MUCH more generic location like lib/util, and ultimately back to Rails NumberHelper.